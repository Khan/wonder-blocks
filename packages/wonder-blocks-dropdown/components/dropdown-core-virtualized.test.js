//@flow
import * as React from "react";
import renderer from "react-test-renderer";

import OptionItem from "./option-item.js";
import DropdownCoreVirtualized from "./dropdown-core-virtualized.js";

describe("DropdownCoreVirtualized", () => {
    it("should render a virtualized list", () => {
        // Arrange
        const items = new Array(100).fill(null).map((item, i) => ({
            component: (
                <OptionItem
                    key={i}
                    value={(i + 1).toString()}
                    label={`School ${i + 1} in Wizarding World`}
                />
            ),
            focusable: true,
            onClick: jest.fn(),
            role: "option",
            populatedProps: {
                selected: false,
                variant: "checkbox",
            },
        }));

        // Act
        const tree = renderer
            .create(
                <DropdownCoreVirtualized
                    data={items}
                    height={360}
                    width={288}
                />,
            )
            .toJSON();

        // Assert
        expect(tree).toMatchSnapshot();
    });
});
