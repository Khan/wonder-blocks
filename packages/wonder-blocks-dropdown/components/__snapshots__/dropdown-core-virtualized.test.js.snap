// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownCoreVirtualized should render a virtualized list 1`] = `
<div
  onScroll={[Function]}
  style={
    Object {
      "WebkitOverflowScrolling": "touch",
      "direction": "ltr",
      "height": 360,
      "overflow": "auto",
      "position": "relative",
      "width": 288,
      "willChange": "transform",
    }
  }
>
  <div
    style={
      Object {
        "height": 4890,
        "pointerEvents": undefined,
        "width": "100%",
      }
    }
  >
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 0,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 1 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 40,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 2 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 80,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 3 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 120,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 4 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 160,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 5 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 200,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 6 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 240,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 7 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 280,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 8 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 320,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 9 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 360,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 10 in Wizarding World
      </span>
    </div>
    <div
      aria-selected="false"
      className=""
      onBlur={[Function]}
      onClick={[Function]}
      onDragStart={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchCancel={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      role="option"
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "border": 0,
          "borderStyle": "solid",
          "borderWidth": 0,
          "boxSizing": "border-box",
          "color": "#21242c",
          "cursor": "default",
          "display": "flex",
          "flexDirection": "row",
          "height": 40,
          "left": 0,
          "margin": 0,
          "minHeight": 40,
          "minWidth": 0,
          "outline": 0,
          "padding": 0,
          "paddingLeft": 8,
          "paddingRight": 16,
          "position": "absolute",
          "top": 400,
          "whiteSpace": "nowrap",
          "width": "100%",
          "zIndex": 0,
        }
      }
      tabIndex={0}
    >
      <div
        className=""
        style={
          Object {
            "alignItems": "stretch",
            "backgroundColor": "#ffffff",
            "borderColor": "rgba(33,36,44,0.50)",
            "borderRadius": 3,
            "borderStyle": "solid",
            "borderWidth": 1,
            "boxSizing": "border-box",
            "display": "flex",
            "flexDirection": "column",
            "margin": 0,
            "minHeight": 16,
            "minWidth": 16,
            "padding": 0,
            "position": "relative",
            "zIndex": 0,
          }
        }
      />
      <span
        className=""
        style={
          Object {
            "MozOsxFontSmoothing": "grayscale",
            "WebkitFontSmoothing": "antialiased",
            "display": "block",
            "fontFamily": "Lato, \\"Noto Sans\\", sans-serif",
            "fontSize": 16,
            "fontWeight": 700,
            "lineHeight": "20px",
            "marginLeft": 8,
            "userSelect": "none",
            "whiteSpace": "nowrap",
          }
        }
      >
        School 11 in Wizarding World
      </span>
    </div>
  </div>
</div>
`;
